src/common/types.ts

```diff
@@ -1,15 +1,21 @@
 import { Ctx, MoveMap, TurnConfig } from "boardgame.io";
 
-export type PlayerIDType = "0" | "1";
-
+export enum PlayerIDType {
+  guesserPlayer = '0',
+  judgePlayer = '1',
+};
```

Comment:
```suggestion
export enum PlayerType {
  Guesser = '0',
  Judge = '1',
};
```
And use it as `PlayerType.Judge`.